packages/terra-action-footer/src/_ActionFooterContainer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import styles from './ActionFooterContainer.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  children: PropTypes.node,
+};
+
+const defaultProps = {
+  children: undefined,
+};
+
+const ActionFooterContainer = ({
+  children,
+  ...customProps
+}) => {
+  const isEmpty = !React.Children.toArray(children).length;
```

Comment:
Interestingly enough, `React.Children.count` _doesn't_ seem to filter out falsey values. Found this when using the terse syntax of the `ActionFooter`.. when no `start` or `end` was given, it was still rendering with the `.with-actions` class..

The fix: use `React.Children.toArray().length` because
>Returns the children opaque data structure as a flat array

See: [reference](https://reactjs.org/docs/react-api.html#reactchildrentoarray)